src-react/containers/path.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux'
 
+import "../styles/styles.scss";
```

Comment:
Oh... I didn't notice that.

BTW, seems that the `react-app` eslint rules allow both `no-semi` and `semi`.
(Check line 2)